pkg/skaffold/deploy/status_check_test.go

```diff
@@ -430,8 +439,11 @@ func TestPrintStatus(t *testing.T) {
 		{
 			description: "multiple resources 1 not complete and in error",
 			rs: []*resource.Deployment{
-				resource.NewDeployment("r1", "test", 1).
-					WithDone("succes", nil),
+				withDone(
+					resource.NewDeployment("r1", "test", 1),
+					"succes",
+					nil,
+				),
 				resource.NewDeployment("r2", "test", 1).
 					WithStatus("", fmt.Errorf("context deadline expired")),
```

Comment:
Looks like `WithStatus` can be moved into the unit test as a helper function as well?

Reply:
yes. i think i will do that in another PR.